orbit/radius.py

```diff
@@ -141,12 +141,6 @@ def mk_cookie_header(self):
 
         return [('Set-Cookie', cookie_val)]
 
-    def __repr__(self):
-        return f'Session({self.token},{self.username},{self.expiry})'
```

Comment:
I'd say leave them in for debugging since it could make errors clearer

Reply:
I don't think that having these has any passive effect. Like the only thing you get when you have an error is the str/repr of the exception object that was raised (but these aren't exception classes) and a stack trace which just lists functions and not stuff like local variables. You would need to add debugging code to like print the object or whatever, but if you are doing that, why not just print the field(s) you actually care about, etc.

Now, to be fair, I originally added this commit to the stack when I was working on coverage and noticed they weren't being hit, but coverage is no longer part of this PR. We can drop this patch if you really want to keep them, but I don't think they are really that useful and stand by removing them.

I am not at my work computer rn, so if you really want to keep these, feel free to rebase this branch and drop `da8ab4d` (shouldn't cause any conflicts) and merge, or you can just merge as is if you can see where I am coming from.

Reply:
sure we can always add them back in if we see a need. I can't say I've ran into a situation where I needed them since September